src/main/java/de/adesso/kicker/team/exception/IdenticalPlayersException.java

```diff
@@ -1,9 +0,0 @@
-package de.adesso.kicker.team.exception;
-
-public class IdenticalPlayersException extends RuntimeException {
-
-    public IdenticalPlayersException() {
-        super("You cannot play with identical players.");
-    }
-
-}
```

Comment:
Maybe still relevant for Matches?

Reply:
Maybe but it doesn't work currently so it's better to remove it for now and reimplement it later.